### ISSUE TITLE
Error when deploying to Heroku in production

### ISSUE THREAD

**Issue by mfratz8686** [colinmeinke/ghost-storage-adapter-s3#27]
Getting the following error when deploying on Heroku:

Unhandled rejection TypeError: _awsSdk2.default.config.setPromisesDependency is not a function
2018-01-31T01:45:37.615073+00:00 app[web.1]:     at new Store (/app/content/adapters/storage/s3/index.js:39:29)
2018-01-31T01:45:37.615074+00:00 app[web.1]:     at Object.getStorage (/app/node_modules/ghost/core/server/adapters/storage/index.js:72:21)
2018-01-31T01:45:37.615076+00:00 app[web.1]:     at setupSiteApp (/app/node_modules/ghost/core/server/web/site/app.js:69:73)
2018-01-31T01:45:37.615077+00:00 app[web.1]:     at setupParentApp (/app/node_modules/ghost/core/server/web/parent-app.js:54:36)
2018-01-31T01:45:37.615079+00:00 app[web.1]:     at /app/node_modules/ghost/core/server/index.js:81:48
2018-01-31T01:45:37.615080+00:00 app[web.1]:     at tryCatcher (/app/node_modules/bluebird/js/release/util.js:16:23)
2018-01-31T01:45:37.615084+00:00 app[web.1]:     at Promise._settlePromiseFromHandler (/app/node_modules/bluebird/js/release/promise.js:512:31)
2018-01-31T01:45:37.615085+00:00 app[web.1]:     at Promise._settlePromise (/app/node_modules/bluebird/js/release/promise.js:569:18)
2018-01-31T01:45:37.615086+00:00 app[web.1]:     at Promise._settlePromise0 (/app/node_modules/bluebird/js/release/promise.js:614:10)
2018-01-31T01:45:37.615087+00:00 app[web.1]:     at Promise._settlePromises (/app/node_modules/bluebird/js/release/promise.js:693:18)
2018-01-31T01:45:37.615088+00:00 app[web.1]:     at Promise._fulfill (/app/node_modules/bluebird/js/release/promise.js:638:18)
2018-01-31T01:45:37.615089+00:00 app[web.1]:     at PromiseArray._resolve (/app/node_modules/bluebird/js/release/promise_array.js:126:19)
2018-01-31T01:45:37.615090+00:00 app[web.1]:     at PromiseArray._promiseFulfilled (/app/node_modules/bluebird/js/release/promise_array.js:144:14)
2018-01-31T01:45:37.615091+00:00 app[web.1]:     at Promise._settlePromise (/app/node_modules/bluebird/js/release/promise.js:574:26)
2018-01-31T01:45:37.615092+00:00 app[web.1]:     at Promise._settlePromise0 (/app/node_modules/bluebird/js/release/promise.js:614:10)
2018-01-31T01:45:37.615093+00:00 app[web.1]:     at Promise._settlePromises (/app/node_modules/bluebird/js/release/promise.js:693:18)

File structure is:

<img width="221" alt="screen shot 2018-01-30 at 9 12 24 pm" src="https://user-images.githubusercontent.com/30666927/35601679-51f6f782-0602-11e8-8488-43b00424774b.png">

Ghost package is included as a npm module. When running locally with the Amazon s3 configuration, everything works. Only seems to be happening when deploying to Heroku.

Any ideas why this is happening and how to solve it?

**Comment by colinmeinke** [comment]
@mfratz8686 did you figure this out?

**Comment by mfratz8686** [comment]
Yes I did. Thanks.

**Comment by danhigham** [comment]
@mfratz8686 Can you share how you fixed the issue, I am seeing the same.

**Comment by mfratz8686** [comment]
@danhigham I am not sure I remember what the issue was exactly. I think it was due to the fact that I had run the initial setup of the database on my local machine but had not run it on the database in Heroku. I found this in the readme I had setup for the project: 

<img width="638" alt="screen shot 2018-03-04 at 4 57 54 pm" src="https://user-images.githubusercontent.com/30666927/36951094-4f6934be-1fcd-11e8-9712-b66912a870cc.png">

I'm really not sure but I hope that helps!
